src/ptvsd/ipcjson.py

```diff
@@ -266,7 +266,7 @@ def process_messages(self):
         while not self.__exit:
             try:
                 self.process_one_message()
-            except AssertionError:
+            except (AssertionError, EOFError):
```

Comment:
I think this will also fix https://github.com/Microsoft/ptvsd/issues/1202 - basically the exception was expected, the problem is that it was incorrectly also logged as error below.